### ISSUE TITLE
Using intent:// URLs on Android always leads to opening the Play Store

### ISSUE THREAD

**Issue by maremmle** [hampusohlsson/browser-deeplink#10]
As far as I understood ``browser-deeplink`` it is supposed to stay on that website if they corresponding app is not installed. But since on Android it will transform the deepening into an intent url, it will actually always open the Google Play Store App as it's apparently listing to those URLs.

**Comment by hampusohlsson** [comment]
Nope, it is not the way it was designed in it's current state. If the app is not installed you get redirected to Google Play, so you can download the app instead.

In a future release I'm looking at changing the behavior and make it more flexible, so you can choose whether or not to redirect :)

**Comment by maremmle** [comment]
Okay I see. I added a switch to turn the entire deeplinking of in android on a fork. @hampusohlsson Could please let me know which minifier you use?

**Comment by hampusohlsson** [comment]
Sure. Any uglifier will do, [perhaps this one](http://skalman.github.io/UglifyJS-online/)

**Comment by danjger** [comment]
This seems to be an issue with which browser is installed on Android. I've see the default (non chrome) browser behave this way but Chrome allows for a fallback url: https://developer.chrome.com/multidevice/android/intents The challenge is to properly handle the cases where the app isn't in the store but you want to take them to a custom location where the APK lives.
